pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool]
```

Comment:
no build-system part?

Reply:
I have not much experience with build systems.
Do you got any idea what to put there?